slurm.spec

```diff
@@ -605,16 +605,6 @@ rm -rf %{buildroot}
 
 %files torque
 %defattr(-,root,root)
-#%{_bindir}/pbsnodes
```

Comment:
deleting it is still a patch on top of upstream slurm.  i would at least replace it with a comment why we don't ship them